events/events-team/projectmanagement/README.md

```diff
@@ -18,7 +21,14 @@ Good communication
 Impartial and objective
 
 # Activities
-Ensure the overall event is a great experience for all contributors Recruit a core team (see best-practices.md) and help recruit other volunteers as necessary
-Create an event strategy with the Content Lead(s) and consider having a social event (decide if social needs it's own lead) Provide venue ideas and suggestions to CNCF; do a walk thru if possible
-Organize and lead n cadence update meetings
-Direct liaison with CNCF Event Manager
\ No newline at end of file
+* Ensure the overall event is a great experience for all contributors Recruit a core team (see best-practices.md) and help recruit other volunteers as necessary.
```

Comment:
Suggest adding a '.' in the middle of "...contributors Recruit..."

Reply:
this too. :)